exercises/luhn-from/src/lib.rs

```diff
@@ -1 +1,18 @@
+pub struct Luhn;
 
+impl Luhn {
+    pub fn is_valid(&self) -> bool {
+        unimplemented!("Determine if the current Luhn struct contains a valid credit card number.");
+    }
+}
+
+/// Here is the example of how the From trait could be implemented
+/// for the &str type. Naturally, you can implement this trait
+/// by hand for the every other type presented in the test suite,
+/// but your solution will fail if a new type is presented.
+/// Perhaps there exists a better solution for this problem?
+impl From<&'static str> for Luhn {
```

Comment:
If we claim we are demoing this implementation for the `&str` type, then we should do it properly: `impl<'a> From<&'a str> for Luhn { ... }`. 

In general, it is bad practice to require the `'static` lifetime. There are exceptions, but they're rare, and not something that students will encounter.